static/app/views/performance/browser/webVitals/pageOverview.spec.tsx

```diff
@@ -10,12 +10,15 @@ import PageOverview from 'sentry/views/performance/browser/webVitals/pageOvervie
 jest.mock('sentry/utils/useLocation');
 jest.mock('sentry/utils/usePageFilters');
 jest.mock('sentry/utils/useOrganization');
+jest.unmock('lodash/debounce');
```

Comment:
👀 why unmock `debounce` here?

Reply:
I was getting a property of undefined error on an upstream component (`SmartSearchBar`) that was using `lodash/debounce`. Seemed to be an issue with the mock, so unmocking fixed the issue.

Anyways, I went back to take a look at this and it looks like `jest.resetAllMocks` is deleting the `debounce` mock, which is why we get the undefined error on the subsequent test. I've removed the `unmock` and replace `resetAllMocks` with `clearAllMocks` which fixes the issue, but cleaner this time.